setup.py

```diff
@@ -106,23 +106,28 @@ def setup_op_geth_repo():
 
 def setup_blockscout_repo():
     github_url = "https://github.com/blockscout/blockscout.git"
-    git_tag = "49eee16ee1078a975a060b1564e6ea5b1ad70f39"
+    git_tag = "b6136977051730a7acd47b35b6bdcf9a74e39be9"
 
     if os.path.isfile("blockscout"):
         raise Exception("Error: 'blockscout' exists as a file and not a directory.")
     elif not os.path.exists("blockscout"):
         print("Cloning the blockscout repository. This may take a while...")
         lib.clone_repo(github_url, "clone the blockscout repository")
 
-        lib.run("checkout stable version", f"git checkout --detach {git_tag}",
+        lib.run("checkout optimism stable version", f"git checkout --detach {git_tag}",
```

Comment:
let's say "checkout blockscout's optmism stable version"

What makes this the "optimism stable version" btw? Could we use a tag instead? Could we include the branch name here in a comment?

Reply:
right now i just use the a specific commit of optimism branch, they don't have a tag for the branch. It is indeed not stable version, however it is fair difficult to get a stable version for our optimism code.

Reply:
Gotcha, let's just add a comment that says `// commit in the "optimism" branch" (use the actual branch name if it's not exactly "optimism")